client/src/pages/Dashboard.js

```diff
@@ -58,26 +78,37 @@ function DashBoard() {
         <div className={classes.mainScreen}>
           <div className={classes.content}>
             <div className={classes.mainHeader}>
-              <Typography>My Mentions</Typography>
-              <ButtonGroup color="primary" style={{ backgroundColor: "white" }}>
-                <Button>Most recent</Button>
-                <Button>Most popular</Button>
-              </ButtonGroup>
+              <Typography className={classes.title}>My mentions</Typography>
+              <ToggleButtonGroup
+                className={classes.toggleButtonsContainer}
+                exclusive
+                size="small"
+                value={order}
+                onChange={handleOrderChange}
+                aria-label="mentions order"
+              >
+                <ToggleButton
+                  className={classes.toggleButtons}
+                  value="most recent"
+                  aria-label="most recent"
+                >
+                  Most recent
+                </ToggleButton>
+                <ToggleButton
+                  className={classes.toggleButtons}
+                  value="most popular"
+                  aria-label="most popular"
+                >
+                  Most popular
+                </ToggleButton>
+              </ToggleButtonGroup>
             </div>
-            <ResultCard />
+            <MentionContainer companyName="tesla" />
```

Comment:
companyName = "tesla"?  is this just skeleton companyName?

Reply:
Yes, it's just a filler. Here we'll pass as props the company name or names.